src/main/java/ch/njol/skript/expressions/ExprHealReason.java

```diff
@@ -25,60 +25,69 @@
 
 import ch.njol.skript.Skript;
 import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Events;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.lang.util.SimpleExpression;
-import ch.njol.skript.log.ErrorQuality;
+import ch.njol.skript.registrations.EventValues;
 import ch.njol.util.Kleenean;
 
-
 @Name("Heal Reason")
-@Description("The <a href='./classes.html#healreason'>heal reason</a> of a heal event. Please click on the link for more information.")
-@Examples({"on heal:",
-	"\tif heal reason = satiated:",
-	"\t\tsend \"You ate enough food and gained health back!\" to player"})
+@Description("The <a href='./classes.html#healreason'>heal reason</a> of a <a href='./events.html#heal'>heal event</a>.")
+@Examples({
+	"on heal:",
+		"\theal reason is satiated",
+		"\tsend \"You ate enough food and gained full health back!\""
+})
+@Events("heal")
 @Since("2.5")
 public class ExprHealReason extends SimpleExpression<RegainReason> {
```

Comment:
Any reason why we don't have event values for this? `heal reason`'s a good expression so I guess people didn't want to duplicate it? I think it might be worth to add them, but tbh not a big deal either way.

Reply:
I did not make this class. It should be an EventValueExpression.